api/filter/accesslog/accesslog.proto

```diff
@@ -32,36 +32,44 @@ message AccessLogCommon {
   // Valid range is (0.0, 1.0].
   double sample_rate = 1 [(validate.rules).double.gt = 0.0, (validate.rules).double.lte = 1.0];
 
-  // This field is the IP and port on which the request from the user was
-  // received.
-  SocketAddress destination_host = 2;
+  // This field is the remote/origin address on which the request from the user was received.
+  // Note: This may not be the physical peer. E.g, if the remote address is inferred from for
+  //       example the x-forwarder-for header, proxy protocol, etc.
+  Address downstream_remote_address = 2;
```

Comment:
Since the overall spec (L22) dictates that all fields refer to "downstream" properties unless otherwise specified, you might be able to omit the `downstream_` prefix here. But it's subjective what's most intuitive, and I don't have strong feelings about it being explicit.